### DISCUSSION TITLE
官网上【PAG 测试素材下载】 -> 【视频转场】资源无法下载

### DISCUSSION THREAD

**Discussion by weizmann** [Tencent/libpag#1658] [Q&A] [answered]
如题，求帮忙提供下视频转场的demo文件吧

感谢！

下载页面
https://pag.art/docs/pag-test-material.html

报错页面
https://pag.art/file/VideoTransmations.zip

报错现场
![image](https://github.com/Tencent/libpag/assets/2306111/d0126edc-9615-41f0-a330-6fd340d3cd0c)

**Comment by kevingpqi123** [comment, marked as answer]
该问题已修复，已更新链接，感谢反馈
